module/Hydro/ranking.py

```
@@ -13,26 +13,33 @@ def fetch_rankings(config: Config) -> list[RankingData]:
     logging.info("开始获取排行榜记录")
     result = []
     page = 1
-    ranking_headers = json_headers.copy()
-    if config.get_config()["session"] is None:
+    ranking_json_headers = json_headers.copy()
+    if "session" not in config.get_config() or config.get_config()["session"] is None:
         raise Exception("登录信息无效，请重试")
```

Ruff (0.14.14)

[warning] Create your own exception

(TRY002)